Leanplum-SDK/Classes/Features/Actions/LPActionManager.m

```diff
@@ -498,4 +533,112 @@ - (void)muteFutureMessagesOfKind:(NSString *)messageId
     }
 }
 
+/**
+ * Checks if message occurrences have reached limits coming from local IAM caps data.
+ * @return True to suppress messages, false otherwise.
+*/
+- (BOOL)shouldSuppressMessages
+{
+    int dayLimit = 0;
+    int weekLimit = 0;
+    int sessionLimit = 0;
+    
+    for (NSDictionary *cap in [[LPVarCache sharedCache] getLocalCaps]) {
+        if (![@"IN_APP" isEqualToString:[cap valueForKey:@"channel"]]) {
+            continue;
```

Comment:
Add logging, for example:
                    LPLog(LPDebug, @"Local IAM caps reached, suppressing messageId=%@", [actionContext messageId]);
                    continue;